DESCRIPTION

```diff
@@ -1,17 +1,14 @@
 Package: hubDevs
 Title: Utilities for creating and standardising Hubverse packages
-Version: 0.0.1
+Version: 0.1.0
 Authors@R: 
```

Comment:
@zkamvar you should add yourself as an author here. And probably someone should be listed as a maintainer.